deployments/ansible/hosts

```diff
@@ -0,0 +1,3 @@
+# Inventory file for production servers
```

Comment:
This file probably shouldn't be here.

Reply:
yes, removed. This was added for test. Ideally the inventory can be passed to playbook from any where by the customer depends on their requirement.